fitbit-react/src/Components/Navbar/Navbar.js

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { GrMenu } from "react-icons/gr";
+import "./navbar.css";
+import SearchBar from "./SearchBar";
+import PrimaryButton from "../Buttons/PrimaryButton";
+import Logo from "./Logo";
+import NavbarIcons from "./NavbarIcons";
```

Comment:
Suggestion: think about a way to reduce the no of lines as in future its going to too long

Reply:
I don't think there is any way to reduce this🤔

Reply:
Done